faults.csv

```diff
@@ -1,63 +1,63 @@
+CSI,Cryo Summary Interlock,"CM Cryo Summary Interlock (He pressure, He Level, Cryo Heartbeat, and CP Ready) faulted","Check if the summary is OK and if yes, reset interlocks",CAV,EDM,$EDM/llrf/rf_srf_intlk_cryo.edl,,,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,CRYO_LTCH,0,,2,Cryo Summary Interlock
+RLK,Resonance/Interlock Link,Resonance/ Interlock Link,,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,RESLINK_LTCH,0,,2,Resonance/Interlock Link
+FPG,FPGA PLL Lock,The LLRF digital clock is no longer guranteed to be locked to the accelerator due to the FPGA detecting a temporary loss of externally supplied LO for the onboard clock,,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,PLL_LTCH,0,,2,FPGA PLL Lock
+IOC,EPICS IOC Watchdog,Interlock trips if the watchdog detects that the IOC is down,Hover over question mark button on screen ,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,IOCWDOG_LTCH,0,,2,EPICS IOC Watchdog
+BLV,Beamline Vacuum,Beamline Vacuum too high,Wait until beamline vacuum is OK and then reset interlocks,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",A,ACCL:{LINAC}:{CRYOMODULE}00:,BMLNVACA_LTCH,0,,2,
+BLV,Beamline Vacuum,Beamline Vacuum too high,Wait until beamline vacuum is OK and then reset interlocks,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",B,ACCL:{LINAC}:{CRYOMODULE}00:,BMLNVACB_LTCH,0,,2,Beamline Vacuum
+CPV,Coupler Vacuum,Coupler manifold vacuum too high,Wait until coupler vacuum is OK and then reset interlocks,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",A,ACCL:{LINAC}:{CRYOMODULE}10:,CPLRVACA_LTCH,0,,2,
+CPV,Coupler Vacuum,Coupler manifold vacuum too high,Wait until coupler vacuum is OK and then reset interlocks,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",B,ACCL:{LINAC}:{CRYOMODULE}10:,CPLRVACB_LTCH,0,,2,Coupler Vacuum
+CP1,Coupler Temperature 1,Temperature sensor on the top of the coupler indicates high temp.,Wait until coupler temperature is OK and then reset interlocks,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,CPLRTEMP1_LTCH,0,,2,Coupler Temperature 1
+CP2,Coupler Temperature 2,Temperature sensor on the bottom of the coupler indicates high temp.,Wait until coupler temperature is OK and then reset interlocks,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,CPLRTEMP2_LTCH,0,,2,Coupler Temperature 2
+STT,Stepper Temperature,Temperature sensor on cavity tuner stepper motor out of tolerance.,Contact SRF on call person,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,STEPTEMP_LTCH,0,,2,Stepper Temperature

```

Comment:
Not sure this fault needs immediate intervention. I think 'take cavity offline and write CATER' would be okay as well